src/Features/ExternalAccess/OmniSharp/Options/OmniSharpSolutionAnalyzerConfigOptionsUpdater.cs

```diff
@@ -0,0 +1,77 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Formatting;
+using Microsoft.CodeAnalysis.ErrorReporting;
+using Microsoft.CodeAnalysis.Options;
+using Roslyn.Utilities;
+using Microsoft.CodeAnalysis.ImplementType;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.OmniSharp.Options;
+
+using Workspace = CodeAnalysis.Workspace;
```

Comment:
why is this needed?

Reply:
Workspace conflicts because there is a `Microsoft.CodeAnalysis.ExternalAccess.OmniSharp.Workspace` namespace.